examples/build/test_boards.py

```diff
@@ -18,6 +18,7 @@
     'xula2': dict(name='led', port='chan', slc=slice(0,4)),
     'xula2_stickit_mb': dict(name='led', port='chan', slc=slice(0,4)),
     'pone': dict(name='led', port='wingA', slc=slice(0,4)),
+    'ppro': dict(name='led', port='wingA', slc=slice(0,4)),
```

Comment:
I caused some confusion by not noticing the `pone` definition used the upper case 'A', 'B', 'C', etc.  Lower-case is preferred, leave it as you changed it but the above needs to reference a valid port in the `ppro`.